install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -51,12 +55,17 @@ data:
     {{- else }}
     #
     # Deprecated: mixer is using EDS
```

Comment:
if you are here, then please remove this outdated comment..

Reply:
done.